.ci/pipeline/build-and-test-lnx.yml

```diff
@@ -24,7 +24,7 @@ steps:
     displayName: "System info"
   - script: |
       conda update -y -q conda
-      conda create -q -y -n CB -c conda-forge -c intel python=$(PYTHON_VERSION) intel::dal-devel mpich pyyaml "dpcpp-cpp-rt=2024.2.0"
+      conda create -q -y -n CB -c conda-forge python=$(PYTHON_VERSION) intel::dal-devel mpich pyyaml "dpcpp-cpp-rt=2024.2.0"

```

Comment:
@ethanglaser  but there is this specific definition to use intel channel anyway - i think it should be removed as well intel::dal-devel?

Reply:
ok, it's in next pr